azure-cli/ad/Add-GroupToServicePrincipal.ps1

```diff
@@ -0,0 +1,33 @@
+function Add-GroupToServicePrincipal(
+    [Parameter(Mandatory = $true)]
+    [EnvironmentConfig] $environmentConfig,
+  
+    [Parameter(Mandatory = $true)]
+    [NamingConfig] $namingConfig,
+
+    [Parameter(Mandatory = $true)]
+    [string] $groupId,
+
+    [Parameter(Mandatory = $false)]
+    [ValidateNotNullOrEmpty()]
+    [string]
+    $serviceInstance = ""
```

Comment:
ValidateNotNullOrEmtpy, Mandatory = $false and default value "" doesn't fit together.
If parameter is optional, Parameter(Mandatory = $false)] should be enough

Reply:
Done